plugins/package-managers/pub/src/test/kotlin/utils/PubCacheReaderTest.kt

```diff
@@ -93,23 +115,23 @@ class PubCacheReaderTest : WordSpec({
             ) shouldBe gitPackageWithPathCacheDir
         }
 
-        "resolve the path of a hosted dependency" {
+        "resolve the path of a custom package repository dependency" {
```

Comment:
Shouldn't this be a new test instead of replacing the existing one?

Reply:
Oh, yes. Fixed copy-paste error.